packages/client/tests/functional/composites-mongo-object/findMany.ts

```diff
@@ -1,50 +1,36 @@
+import { faker } from '@faker-js/faker'
 import pRetry from 'p-retry'
 
-import { getTestClient } from '../../../../../utils/getTestClient'
-import { commentRequiredPropDataA } from '../__helpers__/build-data/commentRequiredPropDataA'
-import { commentRequiredPropDataB } from '../__helpers__/build-data/commentRequiredPropDataB'
+import { setupTestSuite } from './_matrix'
+import { commentDataA, commentDataB } from './_testData'
 
-const describeIf = (condition: boolean) => (condition ? describe : describe.skip)
+// @ts-ignore this is just for type checks
+declare let prisma: import('@prisma/client').PrismaClient
 
-let PrismaClient, prisma
+// can't use faker ids here because the order between generated ids is arbitrary
+const id1 = '8aaaaaaaaaaaaaaaaaaaaaaa'
```

Comment:
That's when `@snaplet/copycat` would be useful :P

Reply:
My thoughts as well :)